.travis.yml

```diff
@@ -15,6 +15,5 @@ install:
   - pip install --upgrade numpy cython
   - pip install -e .
 
-
 script:
-  - true
+  - python setup.py test
```

Comment:
if you add tests with coverage information please enable coveralls or codecov as well

Reply:
So remove the coverage completely or keep it? Also you'd need to enable one of the integrations for the `bio-phys` organization, I can't.

Reply:
sure? Codecov doesn't seem to need any special permissions. You have to change the travis config. Have a look at `MDAnalysis/pytng`

Reply:
Just checked, yes. It's exactly like with Travis-CI. You grant the access for the whole organization once and then all repositories can use it afterwards.

Reply:
should work now